pkg/resource/podtemplatespec.go

```diff
@@ -318,6 +329,32 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, proxyLister configl
 	)
 	deps.AddSecret(defaults.InstallationPullSecret)
 
+	// Project the ServiceAccount token into the Pod (for Pod-identity authentication cases).
+	saVol := corev1.Volume{
+		Name: "bound-sa-token",
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				Sources: []corev1.VolumeProjection{
+					{
+						ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+							Audience: "openshift",
+							Path:     "token",
+						},
+					},
+				},
+			},
+		},
+	}
```

Comment:
I guess that the answer at https://github.com/openshift/cluster-image-registry-operator/pull/640/files#r566756238 will also solve my question regarding this other service account token :-)